airbyte-cdk/bulk/toolkits/extract-cdc/src/main/kotlin/io/airbyte/cdk/read/cdc/DebeziumPropertiesBuilder.kt

```diff
@@ -57,6 +57,8 @@ class DebeziumPropertiesBuilder(private val props: Properties = Properties()) {
         with("value.converter.replace.null.with.default", "false")
         // Timeout for DebeziumEngine's close() method.
         with("debezium.embedded.shutdown.pause.before.interrupt.ms", "10000")
+        // Unblock CDC syncs by skipping errors caused by unparseable DDLs
+        with("schema.history.internal.skip.unparseable.ddl", "true")
```

Comment:
This is indeed the correct Debezium property, but by adding these lines of code here you're setting this property for ALL connectors. Not all connectors maintain a schema history file. That being said, this is just ignored for those connectors, and it seems like this property is named the same way for all those that do maintain a schema history.

Reply:
this is a good point, currently do we have a way for all the connectors to share a common set of debezium properties, while allows specific connectors to include extra properties that won't be shared?